Apps/Examples/Examples/All Examples/Custom3DPuckExample.swift

```diff
@@ -20,12 +20,34 @@ final class Custom3DPuckExample: UIViewController, ExampleProtocol, LocationCons
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-         // The below line is used for internal testing purposes only.
+        // The below line is used for internal testing purposes only.
         finish()
     }
 
-    private func setupExample() {
+    internal func setupExample() {
+        addBuildingExtrusions()
 
+        // set light configuration for shadow visibility
+        var light = Light()
+        light.anchor = .map
+        light.color = StyleColor(.yellow)
+        light.position = [10.0, 40.0, 50.0]
+        light.castShadows = .constant(true)
+        light.shadowIntensity = .constant(0.7)
+        try! mapView.mapboxMap.style.setLight(light)
+
+        let cameraOptions = CameraOptions(center: CLLocationCoordinate2D(latitude: 40.7135, longitude: -74.0066),
+                                          zoom: 15.5,
+                                          bearing: -17.6,
+                                          pitch: 45)
+        mapView.mapboxMap.setCamera(to: cameraOptions)
+
+        // The below lines are used for internal testing purposes only.
+        DispatchQueue.main.asyncAfter(deadline: .now()+5.0) {
+            self.finish()
```

Comment:
Seems to be duplicate to the finish call in view did load

Reply:
good catch, thanks!